pyriemann/datasets/simulated.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+from sklearn.utils.validation import check_random_state
+from .sampling import generate_random_spd_matrix, sample_gaussian_spd
+from pyriemann.utils.distance import distance_riemann
+from pyriemann.utils.base import powm, sqrtm
+
+
+def make_gaussian_blobs(n_samples=100, n_dim=2, class_sep=1.0, class_disp=1.0,
```

Comment:
Like https://scikit-learn.org/stable/modules/generated/sklearn.datasets.make_blobs.html, can you add a parameter `return_centers`?

Reply:
+1